pipeline_api/src/main/java/com/splicemachine/lifecycle/DatabaseLifecycleManager.java

```diff
@@ -255,6 +267,28 @@ private boolean bootServices(State nextState,List<DatabaseLifecycleService> serv
             }
             return true;
         }
+
+        public void setReplicationRole(byte[] role) {
+            if (role == null)
+                return;
+            if (Bytes.compareTo(role, HBaseConfiguration.REPLICATION_NONE) != 0) {
+                if (Bytes.compareTo(role, HBaseConfiguration.REPLICATION_MASTER) == 0) {
+                    setReplicationRoleLocal("MASTER");
+                } else if (Bytes.compareTo(role, HBaseConfiguration.REPLICATION_SLAVE) == 0) {
+                    setReplicationRoleLocal("SLAVE");
+                }
```

Comment:
I think it is a good idea to throw an exception is `role` doesn't match (or at least add an assert)